src/options.ts

```diff
@@ -226,3 +248,7 @@ function getEnvBoolean(key: string, defaultValue = true) {
 
   return true;
 }
+
+function deduplicate(arr: string[]) {
+  return [...new Set(arr)];
```

Comment:
nit: can this be used inline instead of wrapping in a single line, single use function?

Reply:
It could be. The reason I extracted it is because I find this way to be more self-documenting than `[...new Set(propagatorsStr.split(',')))]` would.